ensime_shared/util.py

```diff
@@ -0,0 +1,35 @@
+import os
+from contextlib import contextmanager
+
+class Util:
```

Comment:
creating a general utility class usually just leads to too much clutter everything in here is file specific. A better name for this class is FileOperations or FileUtilities